pavelib/prereqs.py

```diff
@@ -24,10 +24,8 @@
 # a corresponding change to circle.yml, which is how the python
 # prerequisites are installed for builds on circleci.com
 toxenv = os.environ.get('TOXENV')
-if toxenv and toxenv != 'quality-django32':
+if toxenv and toxenv != 'quality':
     PYTHON_REQ_FILES = ['requirements/edx/testing.txt']
-elif toxenv and toxenv == 'quality-django32':
-    PYTHON_REQ_FILES = ['requirements/edx/testing.txt', 'requirements/edx/django.txt']
 else:
     PYTHON_REQ_FILES = ['requirements/edx/development.txt']
```

Comment:
[Author's note] I'm not sure if this was really correct in the first place -- shouldn't the quality checks be using dev deps (pylint will complain about missing django-debug-toolbar) and pytest be using just testing deps?

Reply:
We're looking into the feasibility of ripping out this paver code anyway; note that the GitHub Actions workflows already use pylint directly instead of going through paver.